### ISSUE TITLE
Conflicting dependencies in balancerpool.txt

### ISSUE THREAD

**Issue by AngelaKTE** [TokenEngineeringCommunity/BalancerPools_Model#48]
Error message:
```
"ERROR: Cannot install -r balancerpool.txt (line 42) and python-dateutil==2.8.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested python-dateutil==2.8.1
    ethereum-etl 1.6.1 depends on python-dateutil==2.7.0"
```
----

to solve the issue, I've already tried to:
a) update balancerpool.txt to request `python-dateutil>=2.7.0`
>> caused new error message 
```
The conflict is caused by:
    The user requested python-dateutil>=2.7.0
    ethereum-etl 1.6.1 depends on python-dateutil==2.7.0
    matplotlib 3.3.4 depends on python-dateutil>=2.1
    mythx-models 1.9.3 depends on python-dateutil==2.8.1
```
b) set up a new environment and installed packages from scratch, but failed.
![Bildschirmfoto 2021-02-03 um 10 51 42](https://user-images.githubusercontent.com/42371947/106734396-41d7b700-6613-11eb-86a0-cba10aadab61.png)
![Bildschirmfoto 2021-02-03 um 10 58 43](https://user-images.githubusercontent.com/42371947/106734416-48662e80-6613-11eb-8bc6-ba950fa1e933.png)

**Comment by AngelaKTE** [comment]
also tried variants in balancerpool.txt:

`python-dateutil~=2.7.0` 
`python-dateutil` 

...but didn't solve the issue.

**Comment by AngelaKTE** [comment]
Solved with using updated requirements.txt
after Merged #47 into dev.

**Comment by AngelaKTE** [comment]
Proposal to avoid confusion in documentation:

rename "data/requirements.txt" to "data/datapull-requirements.txt"
to avoid confusion with "requirements.txt" in top-level repo
